Framework/PythonInterface/mantid/_plugins/src/Exports/ProductFunction.cpp

```diff
@@ -0,0 +1,24 @@
+#include "MantidCurveFitting/Functions/ProductFunction.h"
+#include <boost/python/class.hpp>
+#include <boost/python/overloads.hpp>
+
+using Mantid::CurveFitting::Functions::ProductFunction;
+using Mantid::API::CompositeFunction;
+using namespace boost::python;
+
+namespace {
```

Comment:
These functions are not needed so we should be able to delete the whole anonymous namespace.